src/modules/mana/sagas.ts

```diff
@@ -434,10 +470,10 @@ function* sendWalletMetaTransaction(
     contractConfig.abi,
     metaTxChainProvider
   )
-  const tx = yield call(() =>
+  const tx: { data: string } = yield call(() =>
```

Comment:
This type should be `{ data: string | null }` to be correct. We can leave it as it is, but we should remove the `!` from the `tx.data!` a few lines below.

Reply:
Removed `!`,  thanks!